merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/services/GetSimulationResultsAction.java

```diff
@@ -76,7 +78,7 @@ public Map<String, List<Pair<Duration, SerializedValue>>> getResourceSamples(fin
     final var revisionData = this.planService.getPlanRevisionData(planId);
     final var simulationResults$ = this.simulationService.get(planId, revisionData);
     if (simulationResults$.isEmpty()) return Collections.emptyMap();
-    final var simulationResults = simulationResults$.get();
+    final var simulationResults = simulationResults$.get().getSimulationResults();
```

Comment:
`simulationResults` needs to be renamed if it is no longer a `SimulationResults`